### DISCUSSION TITLE
Restart bug

### DISCUSSION THREAD

**Discussion by Matssa56** [isc30/ryzen-gpu-passthrough-proxmox#2] [Q&A] [answered]
Hey,

Thanks for the update :)

Concerning the amd-reset bug, is the script still needed? From what I was the issue was solved with the new file.

**Comment by isc30** [comment, marked as answer]
Hey, I think it's not needed since the "vendor-reset" doesn't support the 680M.

I can see improvements with AmdResetBugFix.exe, that one seems to help.

Still facing the reset issue when stopping a VM, next time I try to start it I get the "error 43" and need to restart the host to get it working again.

I can edit the guide to remove the "vendor-reset" part, but I'm hoping they release a new version with the 680M support and starts working

**Reply by isc30** [reply]
just updated the guide, I also found this newer version of the RadeonResetBugFix that seems to work even better (and works in in win11 too)
https://oomza.cutegay.software/inga-lovinde/RadeonResetBugFix/releases

**Comment by isc30** [comment]
I just did some progress on this, I'm able to start a VM with GPU passthrough working. Then shutdown the VM, then start it again and GPU passthrough works! It seems like the original amdgpu driver is helping with the reset. I updated the guide to load vfio without blacklisting the amd drivers, which seems like a nice combination for stability

**Comment by Mpaulo** [comment]
Hi guys, I confirm that it works on the 7950x3d, along with a 6900xt, both run on 2 VMs (1 for gaming and another for work). 
However, I have a problem with the iGPU as it does not return video to the monitor. Sometimes i need to use remote acess and reinstall adrenaline again, after thata the video shows up, but i need to keep doing on every system restar?. Theres anyway to avoid this thing?

**Reply by isc30** [reply]
That sounds like the reset issue, its a massive pain to restart the host after using the GPU. I haven't found any solution for it yet... it's on AMD or QEMU side I think...
